amd/src/learningmap.js

```diff
@@ -373,11 +383,78 @@ export const init = () => {
          * @param {*} evt
          */
         function endDrag(evt) {
-            evt.preventDefault();
+            if (evt.cancelable) {
+                evt.preventDefault();
+            }
             selectedElement = null;
             unselectAll();
             updateCode();
         }
+
+        /**
+         * Function called when touchstart event occurs.
+         * @param {*} evt
+        */
+        function startTouch(evt) {
+            if (evt.cancelable) {
+                evt.preventDefault();
+            }
+            if (evt.target.classList.contains('learningmap-draggable')) {
+                if (!touchstart) {
+                    touchstart = true;
+                    touchmove = 0;
+                    touchend = false;
+                    setTimeout(
+                        (evt) => {
+                            if (touchmove < 3 && !touchend) {
+                                showContextMenu(evt);
+                            }
+                        },
+                        2000,
+                        evt
```

Comment:
is this additional param needed?

Reply:
To hand over the event to the called function as a parameter

Reply:
Yes, I know what the param means, I just wanted to ask, if it's needed there. But I tested (mainly out of personal interest ;-)) and you're totally right. It has to be passed there to enable the arrow function to pass it onto `showContextMenu`. Sorry for asking :-)